src/app/settings/panel-settings/components/create-panel-layout-configuration/create-panel-layout-configuration.component.ts

```diff
@@ -0,0 +1,103 @@
+import { Component, EventEmitter, OnInit, Output } from '@angular/core'
+import { PanelLayoutConfiguration } from '../../models/panel-layout-configuration'
+import { FormGroup, NonNullableFormBuilder, Validators } from '@angular/forms'
+import { PanelModel, PanelType, SkaarhojModel } from '../../models/panel-enums'
+import { SelectOption } from '../../../../shared/models/select-option'
+
+@Component({
+  selector: 'sofie-create-panel-layout-configuration',
+  templateUrl: './create-panel-layout-configuration.component.html',
+})
+export class CreatePanelLayoutConfigurationComponent implements OnInit {
+  @Output()
+  public onSave: EventEmitter<PanelLayoutConfiguration> = new EventEmitter()
+  @Output()
+  public onCancel: EventEmitter<void> = new EventEmitter()
+
+  public form: FormGroup
+
+  public typeOptions: SelectOption<PanelType>[] = [
+    {
+      name: PanelType.SKAARHOJ,
+      value: PanelType.SKAARHOJ,
+    },
+  ]
+
+  public modelOptions: SelectOption<PanelModel>[] = []
+
+  public saveButtonTooltip?: string
+
+  constructor(private readonly formBuilder: NonNullableFormBuilder) {}
+
+  public ngOnInit(): void {
+    this.form = this.formBuilder.group({
+      type: this.formBuilder.control(PanelType.SKAARHOJ, Validators.required),
+      model: this.formBuilder.control('', Validators.required),
+      name: this.formBuilder.control('', Validators.required),
+      inputConfigurations: this.formBuilder.control({}),
+    })
+
+    this.form.get('type')?.valueChanges.subscribe((panelType: PanelType) => this.updateModelOptions(panelType))
+    this.updateModelOptions(PanelType.SKAARHOJ)
+
+    this.form.valueChanges.subscribe(() => this.updateSaveButtonTooltip())
+    this.updateSaveButtonTooltip()
+  }
+
+  private updateModelOptions(panelType: PanelType): void {
+    switch (panelType) {
+      case PanelType.SKAARHOJ: {
+        this.modelOptions = [
+          {
+            name: SkaarhojModel.MK48,
+            value: SkaarhojModel.MK48,
+          },
+          {
+            name: SkaarhojModel.MKT1A,
+            value: SkaarhojModel.MKT1A,
+          },
+        ]
+        break
+      }
+      default: {
+        this.modelOptions = []
+      }
+    }
+  }
+
+  public save(): void {
```

Comment:
I think you need this in here as well, since you want to be able to disable the button :)

```suggestion
  public save(): void {
  if(!form.valid) {
    return
  }
```